Casks/o/orka-desktop.rb

```diff
@@ -0,0 +1,21 @@
+cask "orka-desktop" do
+  version "3.0.0-beta.1"
+  sha256 "73979664b53c186e28ce83962639151a63c50be3fdddad2348c7a634bdfd85d8"
+
+  url "https://github.com/macstadium/orka-desktop/releases/download/#{version}/OrkaDesktop-#{version}.dmg"
+  name "Orka Desktop (Beta)"
+  desc "Run macOS virtual machines locally and build images for use with Orka"
+  homepage "https://github.com/macstadium/orka-desktop"
+
+  livecheck do
+    url :url
+    regex(/v?(\d+(?:\.\d+)+-beta\.\d+)/i)
```

Comment:
```suggestion
    regex(/v?(\d+(?:\.\d+)+(?:-beta\.\d+)?)/i)
```

This should continue to work once releases are stable, however we can also likely remove the `regex` at that time.

Reply:
Thanks for the suggestion! I went on and updated the regex